app/src/main/java/com/markkolenbrander/capstonenewsapp/Article.kt

```diff
@@ -0,0 +1,12 @@
+package com.markkolenbrander.capstonenewsapp
+
+data class Article(
+//    val source: Source,
```

Comment:
Just curious, why have this commented out? Since you have the `Source` class and also sample sources created in the MainActivity, you could leave this one uncommented.

Reply:
Yes that's true I had it commented out for testing and I hadd to uncomment it still. Thanks